pages/room/index.js

```diff
@@ -27,9 +29,21 @@ const CreateRoom = ({ spotifyAuthURL, spotifyCode }) => {
     const session = await getSession();
     const { name, email } = session.user;
     const result = await axios.get('/api/user/get-user', { params: { name, email } });
+    const GET_USERS = gql`
+      query {
+        users(name: name, email: email) {
+          name
+          email
+          image
+        }
+      }
+    `;
+    const { loading, error, data } = useQuery(GET_USERS);
+    console.log('data', data);
     const user = result.data.user;
 
     const newClient = { ...session, user };
+    console.log(newClient, user, 'session', session, 'id', user._id);
```

Comment:
console logs. Please check your prs after you make them to avoid these errors please.